pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/TableRebalancer.java

```diff
@@ -175,67 +166,127 @@ public RebalanceResult rebalance(TableConfig tableConfig, Configuration rebalanc
     if (currentAssignment.equals(targetAssignment)) {
       LOGGER.info("Table: {} is already balanced", tableNameWithType);
       if (reassignInstances) {
-        return new RebalanceResult(RebalanceResult.Status.DONE, "Instance reassigned, table is already balanced",
-            instancePartitionsMap, targetAssignment);
+        if (dryRun) {
+          return new RebalanceResult(RebalanceResult.Status.DONE,
+              "Instance reassigned in dry-run mode, table is already balanced", instancePartitionsMap,
+              targetAssignment);
+        } else {
+          return new RebalanceResult(RebalanceResult.Status.DONE, "Instance reassigned, table is already balanced",
+              instancePartitionsMap, targetAssignment);
+        }
       } else {
         return new RebalanceResult(RebalanceResult.Status.NO_OP, "Table is already balanced", instancePartitionsMap,
             targetAssignment);
       }
     }
 
     if (dryRun) {
-      LOGGER.info("Rebalance table: {} in dry-run mode, returning the target assignment", tableNameWithType);
+      LOGGER.info("Rebalancing table: {} in dry-run mode, returning the target assignment", tableNameWithType);
```

Comment:
```suggestion
      LOGGER.info("Rebalancing table {} in dry-run mode, returning the target assignment", tableNameWithType);
```

Reply:
I prefer having a `:` before the variables so it is easier to locate the variables